File: project/jimmer-sql/src/test/java/org/babyfish/jimmer/sql/model/flat/Country.java
package org.babyfish.jimmer.sql.model.flat;

import org.babyfish.jimmer.sql.*;

import java.util.List;

@Entity
@Table(name = "FLAT_COUNTRY")
@DatabaseValidationIgnore
public interface Country {

    @Id
    long id();

    String countryName();

    @OneToMany(mappedBy = "country")
    List<Province> provinces();
}

File: project/jimmer-sql-kotlin-test-model/src/main/kotlin/org/babyfish/jimmer/sql/kt/model/flat/Country.kt
package org.babyfish.jimmer.sql.kt.model.flat

import org.babyfish.jimmer.sql.DatabaseValidationIgnore
import org.babyfish.jimmer.sql.Entity
import org.babyfish.jimmer.sql.Id
import org.babyfish.jimmer.sql.Table

@Entity
@Table(name = "FLAT_COUNTRY")
@DatabaseValidationIgnore
interface Country {

    @Id
    val id: Long

    val countryName: String
}